### PR TITLE
🐛(frontend) fix editor height and translations

### PR DESCRIPTION
**Pull request by PanchoutNathan**
## Purpose

The size of the doc editor is not full height

### CODE REVIEW

File: src/frontend/apps/impress/src/features/docs/docs-grid/components/DocsGridItem.tsx [hunk 1]
```diff
@@ -10,6 +10,16 @@ import { Doc, LinkReach } from '../../doc-management';
 import { DocsGridActions } from './DocsGridActions';
 import { SimpleDocItem } from './SimpleDocItem';
 
+const ellipsisCss = css`
```
Comment (AntoLC) [inline src/frontend/apps/impress/src/features/docs/docs-grid/components/DocsGridItem.tsx:13]: Do you use it, I don't see it ?

File: src/frontend/apps/impress/src/i18n/translations.json [hunk 2]
```diff
@@ -257,29 +225,29 @@
...
-      "You don't have any document yet.": "Vous n'avez pas encore de document.",
       "Your current document will revert to this version.": "Votre document actuel va revenir à cette version.",
-      "Your role": "Votre rôle",
-      "Your role:": "Votre rôle:",
       "Your {{format}} was downloaded succesfully": "Votre {{format}} a été téléchargé avec succès",
       "accessibility-contact-defenseurdesdroits": "Contacter le délégué du<1>Défenseur des droits dans votre région</1>",
       "accessibility-dinum-services": "<strong>DINUM</strong> s'engage à rendre accessibles ses services numériques, conformément à l'article 47 de la loi n° 2005-102 du 11 février 2005.",
       "accessibility-form-defenseurdesdroits": "Écrire un message au<1>Défenseur des droits</1>",
       "accessibility-not-audit": "<strong>docs.numerique.gouv.fr</strong> n'est pas en conformité avec le RGAA 4.1. Le site n'a <strong>pas encore été audité.</strong>",
+      "doc title input": "entrée titre doc",
```
Comment (AntoLC) [inline src/frontend/apps/impress/src/i18n/translations.json:250]: It is not something dedicated for the tests ?

File: src/backend/locale/de_DE/LC_MESSAGES/django.po
Comment (AntoLC) [inline src/backend/locale/de_DE/LC_MESSAGES/django.po:1]: Don't versioned it in this PR, it is for the backend, it is often only the date and hours that change.

File: src/backend/locale/en_US/LC_MESSAGES/django.po
Comment (AntoLC) [inline src/backend/locale/en_US/LC_MESSAGES/django.po:1]: Same, don't version it.

File: src/backend/locale/fr_FR/LC_MESSAGES/django.po
Comment (AntoLC) [inline src/backend/locale/fr_FR/LC_MESSAGES/django.po:1]: Same, don't version it.

File: src/frontend/apps/impress/src/i18n/translations.json (hunk 2 above)
Comment (PanchoutNathan) [reply src/frontend/apps/impress/src/i18n/translations.json:250]: Yes not wrong, what do you want to do? Should we remove the trad and put in hard?

**Review by AntoLC — APPROVED** [review]

**Pull request merged**